src/irmin/hash.mli

```diff
@@ -42,7 +41,10 @@ module BLAKE2B : S.HASH
 
 module BLAKE2S : S.HASH
 
+(** v1 serialisation *)
+module V1 (H : S.HASH) : S.HASH with type t = H.t
+
+(** Typed hashes. *)
+
```

Comment:
extra space?

Reply:
I was preserving the attachment as it was before (https://github.com/mirage/irmin/pull/926/files/024f60809bb401c909290c78d3d75c2c778fb507#diff-1be34172140401295ad7258b1add6782L1104), but it's probably better fixed onto the `Make` functor anyway.

Reply:
Fixed in https://github.com/mirage/irmin/pull/926/commits/abd91f4dbe21be37f35a6b7edb524f1c51e1405f.